src/components/SQFormDialogStepper/SQFormDialogStepper.tsx

```diff
@@ -113,14 +165,17 @@ export function SQFormDialogStepper({
     setCompleted([...completed, activeStep]);
   };
 
-  const handleStep = step => () => {
+  const handleStep = (step: number) => () => {
     const nextStep = step;
-    if ([nextStep].some(step => completed.includes(step))) {
+    if ([nextStep].some((step) => completed.includes(step))) {
       setActiveStep(step);
     }
   };
 
-  const handleSubmit = async (values, helpers) => {
+  const handleSubmit = async (
+    values: Record<string, unknown>,
+    helpers: unknown
```

Comment:
The type for these helps we can get from formik. `FormikBag` is the type and it'll take two generics. The first will likely just be `unknown` and the second will be the generic this component takes in indicating the values of the form mentioned in a previous comment.